### PR TITLE
etching_rate_of_target_in_magnetron

### PR DESCRIPTION
**Pull request by Zulusssss**
add law and tests

### CODE REVIEW

File: symplyphysics/laws/chemistry/etching_rate_of_target_in_magnetron.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+from sympy.physics.units import elementary_charge, avogadro_constant
+from symplyphysics import (units, Quantity, Symbol, validate_input, validate_output,
+                           dimensionless,)
+
+# Description
+## The ions of the gas-discharge plasma in the magnetron fall on the target and knock the atoms out of it.
+## The sputtering coefficient shows how many target atoms are knocked out of the target by a single ion.
+
+## Law is: V =  J * M * Y / (q * po * Na), where
+## V - target etching rate,
```
Comment (blackyblack) [inline symplyphysics/laws/chemistry/etching_rate_of_target_in_magnetron.py:11]: What's this?

File: symplyphysics/laws/chemistry/etching_rate_of_target_in_magnetron.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## J - density of the current of the ion flux incident on the target,
```
Comment (blackyblack) [inline symplyphysics/laws/chemistry/etching_rate_of_target_in_magnetron.py:12]: Does angle matter?

File: symplyphysics/laws/chemistry/etching_rate_of_target_in_magnetron.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+## M - mass of the target atom,
+## Y - sputtering coefficient,
+## q - elementary charge,
+## po - target density,
+## Na - avogadro constant.
+
+etching_rate = Symbol("etching_rate", units.velocity)
+
+ion_current_density = Symbol("ion_current_density", units.current / units.area)
+mass_of_target_atom = Symbol("mass_of_target_atom", units.mass / units.amount_of_substance)
```
Comment (blackyblack) [inline symplyphysics/laws/chemistry/etching_rate_of_target_in_magnetron.py:22]: Molar mass, I guess.

File: test/chemistry/etching_rate_of_target_in_magnetron_test.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+## is 47.867 kilogram per mole. The atomization coefficient of the target atoms is 1.873. The target density is
+## 6020 [kilogram / meter^3]. Then the etching rate of the target is 15.06 nanometer per second.
+
+Args = namedtuple("Args", ["ion_current_density", "mass_of_target_atom", "sputtering_coefficient", "target_density"])
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    ion_current_density = Quantity(97.566 * units.ampere / units.meter**2)
+    mass_of_target_atom = Quantity(47.867 * units.gram / units.mol)
```
Comment (blackyblack) [inline test/chemistry/etching_rate_of_target_in_magnetron_test.py:17]: Does not correspond to description.

File: symplyphysics/laws/chemistry/etching_rate_of_target_in_magnetron.py (hunk 2 above)
Comment (Zulusssss) [reply symplyphysics/laws/chemistry/etching_rate_of_target_in_magnetron.py:12]: > Does angle matter?

As I understand it, in this case it does not depend.

File: symplyphysics/laws/chemistry/etching_rate_of_target_in_magnetron.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
+from sympy import Eq, solve
+from sympy.physics.units import elementary_charge, avogadro_constant
+from symplyphysics import (units, Quantity, Symbol, validate_input, validate_output,
+                           dimensionless,)
+
+# Description
+## The ions of the gas-discharge plasma in the magnetron fall on the target and knock the atoms out of it.
+## The sputtering coefficient shows how many target atoms are knocked out of the target by a single ion.
+## The etching rate is how many nanometers of the target substance are etched per unit of time. In other words,
+## this is how much thinner a nanometer becomes a target per unit of time.
```
Comment (blackyblack) [inline symplyphysics/laws/chemistry/etching_rate_of_target_in_magnetron.py:10]:
```suggestion
## this is how much thinner a target becomes per unit of time.
```

**Review by blackyblack — APPROVED** [review]

**Pull request merged**